### PR TITLE
Personnalize Name Format with shibboleth 1.3

### PR DESCRIPTION
**Pull request by cdesbouche-acipia**
Like title says.

### CODE REVIEW

File: lib/SimpleSAML/XML/Shib13/AuthnResponse.php [hunk 1]
```diff
@@ -322,11 +322,28 @@ public function generate(SimpleSAML_Configuration $idp, SimpleSAML_Configuration
...
+		} else {
+                	if (!array_key_exists($nameidattribute, $attributes)) {
+                        SimpleSAML_Logger::error('Unable to add NameID: Missing ' . var_export($nameidattribute, TRUE) .
+                                ' in the attributes of the user.');
+                        return NULL;
+                	}
+			$nameid = $attributes[$nameidattribute][0] ;
+                        SimpleSAML_Logger::debug('NameID: ' . $nameidattribute . ' = ' . $nameid ) ;
+		}
+		$nameidformat = $sp->getString('NameIDFormat', 'xurn:mace:shibboleth:1.0:nameIdentifier');
```
Comment (jaimeperez) [inline lib/SimpleSAML/XML/Shib13/AuthnResponse.php]: I believe there is a typo here and instead of `xurn` it should read just `urn`, right?

**Comment by jaimeperez** [comment]
Thanks for the PR @cdesbouche-acipia, and sorry so much for the delay!

I guess this is more a bugfix than a new functionality, right? In that case, would it be possible to rebase this on top of master, instead of the 1.12 branch?

Thanks again!

**Comment by tvdijen** [comment]
If this is still a desirable change, the rebase can be found here:
https://github.com/tvdijen/simplesamlphp/commit/9ed2f7fc629778c9b5cc611131e832dd7727d7bd

**Comment by tvdijen** [comment]
Closing this as Shib13 is deprecated and will be removed in 2.0

**Pull request closed without merging**